### ISSUE TITLE
Help to combine mediapipe holistic with iris

### ISSUE THREAD

**Issue by Hakazaba** [google/mediapipe#1546]
Hello, could i please have some help combining mediapipe holistic with iris? i need to do both at the same time in the application i'm developing. I also find the blink detection of iris essential with any face tracking to correctly detect blinking.

**Comment by hadon** [comment]
I'll mention contributor @grishchenko-ivan, who may have some advice.

**Comment by ivan-grishchenko** [comment]
Hi there!

TL;DR You'll need to use predicted face landmarks to locate eye regions and run iris model for them.

To begin with you'll need to create a subgraphs similar to [FaceLandmarksFromPoseGpu](https://github.com/google/mediapipe/blob/master/mediapipe/modules/holistic_landmark/face_landmarks_from_pose_gpu.pbtxt). But you'll name it IrisLandmarksFromFaceGpu. Also check nested subgraphs.

To understand how to crop eye region from predicted face landmarks, take a look here: https://github.com/google/mediapipe/blob/master/mediapipe/graphs/iris_tracking/iris_tracking_gpu.pbtxt

Finally add your IrisLandmarksFromFaceGpu after the face prediction in Holistic (here: https://github.com/google/mediapipe/blob/master/mediapipe/modules/holistic_landmark/holistic_landmark_gpu.pbtxt#L110)

**Comment by ivan-grishchenko** [comment]
:thumbsup:

**Comment by ivanmang** [comment]
I have used the calculator provided here: https://github.com/google/mediapipe/pull/1559

and trying to add add the _IrisLandmarkLeftAndRightFromFaceLandmarksGpu_ after the face prediction in Holistic.

However, seems like it still doesn't work. The app crashes after a few second on my Samesung S21 phone.
I stuck here for a few days and would appreciate if you can spot the mistake.

Here is my implementation:
https://github.com/ivanmang/-mediapipe

**Comment by sgowroji** [comment]
Hi @Hakazaba, Could you please respond if you still looking resolution for the above query. Thanks!
